app/services/analytics_events.rb

```diff
@@ -1549,6 +1554,7 @@ def idv_front_image_clicked(
       source: source,
       use_alternate_sdk: use_alternate_sdk,
       liveness_checking_required: liveness_checking_required,
+      captureAttempts: captureAttempts,
```

Comment:
👍🏿 ... i think this is also needed for idv_selfie_image _clicked as well

Reply:
@amirbey added.